build.sbt

```diff
@@ -44,7 +44,7 @@ ReleaseKeys.publishArtifactsAction := PgpKeys.publishSigned.value
 val curatorVersion = "2.10.0"
```

Comment:
Could you please also update curator version (and maybe also other dependencies) to version from scala-util?